src/vuestic-theme/vuestic-components/vuestic-checkbox/VuesticCheckbox.vue

```diff
@@ -1,8 +1,17 @@
 <template>
-    <div class="form-check abc-checkbox abc-checkbox-primary" :class="{'abc-checkbox-circle': circle}">
-        <input class="form-check-input" :id="id" type="checkbox" :checked="checked" @change="onChange" :disabled="disabled">
+    <div class="form-check abc-checkbox" :class="classArray">
+        <input class="form-check-input" :id="id" type="checkbox" :checked="checked" @change="onChange"
+               :disabled="disabled">
         <label class="form-check-label" :for="id">
-            <span class="abc-label-text">{{label}}</span>
+            <template v-if="label">
+                <span class="abc-label-text" v-html="label"></span>
```

Comment:
I don't think you need v-html here - you have a slot for that. Just use {{:label}} here